examples/adder/src/lib.rs

```diff
@@ -5,7 +5,7 @@ use near_sdk::{env, near, Promise};
 pub struct Pair(u32, u32);
 
 #[derive(Default)]
-#[near(serializers=[borsh, json], contract_state)]
+#[near(serializers=[borsh, json], contract_state(key = b""))]
```

Comment:
Added is one of the simplest examples, please, don't complicate it. Let's revert the change here. If you want to showcase it, please, make a new example.